src/Simulation/EntryPointDriver/IEntryPoint.cs

```diff
@@ -33,6 +33,14 @@ public interface IEntryPoint
         /// </summary>
         IEnumerable<Option> Options { get; }
 
+        /// <summary>
+        /// Generates payload for Azure Quantum for the entry point.
+        /// </summary>
+        /// <param name="parseResult">The command-line parsing result.</param>
+        /// <param name="settings">The generate Azure payload settings.</param>
+        /// <returns>The exit code.</returns>
+        Task<int> GenerateAzurePayload(ParseResult parseResult, GenerateAzurePayloadSettings settings);
```

Comment:
> /// <returns>The exit code.</returns>

The return type `Task<int>` does not look like exit code. If that's ok then the explanation "The exit code" is still not very informative. Some more details wouldn't harm (which values in which cases).

Reply:
I expanded the description of the return value for all methods in this interface.

Reply:
Regarding the values themselves, it is not really a fixed set of values, hence we can't list them all. And since this is an interface, the implementation details is up to the interface implementer. The integer value is just for callers and implementers to know whether the operation was successful.

Reply:
Actually, we do have a known value for the exit code, `0` means success. I'll update the documentation to add this detail. Thanks Robin!

Reply:
Updated.